include/xtensor-sparse/xlil.hpp

```diff
@@ -41,8 +59,8 @@ namespace xt
     private:
 
         using index_type = size_type;
-        using list_index_type = std::vector<index_type>;
-        using container_type = std::map<list_index_type, std::map<index_type, value_type>>;
+        using list_index_type = svector<size_type>;
+        using container_type = std::map<list_index_type, value_type>;
```

Comment:
I think that we can remove this definition of `index_type` and rename `list_index_type` into `index_type`